xLights/effects/RipplePanel.cpp

```diff
@@ -203,31 +372,60 @@ RipplePanel::RipplePanel(wxWindow* parent) : xlEffectPanel(parent)
 	FlexGridSizer57->Add(-1,-1,1, wxALL|wxALIGN_CENTER_HORIZONTAL|wxALIGN_CENTER_VERTICAL, 5);
 	FlexGridSizer57->Add(-1,-1,1, wxALL|wxALIGN_CENTER_HORIZONTAL|wxALIGN_CENTER_VERTICAL, 5);
 	SetSizer(FlexGridSizer57);
+	FlexGridSizer57->Fit(this);
+	FlexGridSizer57->SetSizeHints(this);
 
 	Connect(ID_CHOICE_Ripple_Object_To_Draw,wxEVT_COMMAND_CHOICE_SELECTED,(wxObjectEventFunction)&RipplePanel::OnChoice_Ripple_Object_To_DrawSelect);
```

Comment:
These are all present below outside the generated section.  wxSmith kept forgetting about the action handlers in the .wxs file and propogating that, so putting them outside the generated section seemed much more reliable.